starwars/src/components/CharacterComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+
+
+const Character = ({ info }) => {
```

Comment:
Don't need braces around info here. Info is an object already, so you're essentially putting braces around braces.